bftengine/src/bftengine/CollectorOfThresholdSignatures.hpp

```diff
@@ -351,7 +351,7 @@ class CollectorOfThresholdSignatures {
       // TODO(GG): can utilize several threads (discuss with Alin)
 
       const uint16_t bufferSize = (uint16_t)verifier->requiredLengthForSignedData();
-      char* const bufferForSigComputations = (char*)alloca(bufferSize);  // TODO(GG): check
+      std::vector<char> const bufferForSigComputations(bufferSize);
```

Comment:
The const used here is for the pointer itself, not for the memory, which it points to. Remove the const, because it makes the vector itself const.

Reply:
Fixed.